backend/monitoring/dashboard.py

```
@@ -746,5 +1108,520 @@ async def websocket_endpoint(websocket: WebSocket):
+            print(f"Error fetching session metrics: {metrics_error}")
+        
+        # Check frontend components availability
+        frontend_status = {"available": True, "components": {}}
+        frontend_components = ['LiveChat.jsx', 'InteractiveAudioChat.jsx', 'AgoraVideoCall.jsx']
+        
+        for component in frontend_components:
+            try:
+                import os
+                component_path = os.path.join('frontend', 'src', 'components', component)
+                frontend_status["components"][component] = os.path.exists(component_path)
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)